x/proof/keeper/msg_server_create_claim.go

```diff
@@ -15,9 +15,6 @@ func (k msgServer) CreateClaim(
 	ctx context.Context,
 	msg *types.MsgCreateClaim,
 ) (_ *types.MsgCreateClaimResponse, err error) {
-	// TODO_BLOCKER(@bryanchriswhite): Prevent Claim upserts after the ClaimWindow is closed.
-	// TODO_BLOCKER(@bryanchriswhite): Validate the signature on the Claim message corresponds to the supplier before Upserting.
```

Comment:
This is guaranteed by the presence of the the `cosmos.msg.v1.signer` option on the `MsgCreateClaim` protobuf type.